gpustat/core.py

```diff
@@ -259,10 +267,13 @@ def __init__(self, gpu_list, driver_version=None):
         self.driver_version = driver_version
 
     @staticmethod
-    def new_query():
+    def new_query(include_power=True):
```

Comment:
If the `include_power` is for speed issue, it's better to add some comments to explain.